### DISCUSSION TITLE
Hide login button and side navigation

### DISCUSSION THREAD

**Discussion by begovic92** [oqtane/oqtane.framework#1724] [Q&A]
Hello
I would like to thank creators of this brilliant software. I have a couple of questions:

1. Is it possible to hide login button through configuration, logout should be still shown when logged in
2. Is there a way to add reusable side navigation without building a new module

Thanks in advance

**Comment by leigh-pointer** [comment]
@begovic92 Have a look at the Themes folder in the Oqtane.Client folder.
The controls for the Themes are located at oqtane.framework\Oqtane.Client\Themes\Controls\Theme\ 
You are able to create you own Themes using the framework controls or create your own controls for your theme.

**Comment by sbwalker** [comment]
The Login theme control has a parameter named ShowLogin which you can set to false if you do not want the login button to be displayed. The default Oqtane theme has Theme Settings which are displayed in the Edit Page dialog where you can set the Show Login option interactively - otherwise you would need to create a custom theme.

The default Blazor theme uses a side navigation menu which is a reusable component that in the folder that @leigh-pointer  mentioned above.

**Reply by Almir-Begovic** [reply]
Thank you guys.

I'll make some tweaks to the theme and should be fine. I have one more question. Is there any guide how to localise the site for multiple languages

**Comment by leigh-pointer** [comment]
> Is there any guide how to localise the site for multiple languages

This is a repository for this [https://github.com/oqtane/oqtane.translations](https://github.com/oqtane/oqtane.translations)

**Reply by Almir-Begovic** [reply]
Brilliant, thank you.
